packages/analytics-types/src/diagnostic-event.ts

```diff
@@ -0,0 +1,10 @@
+export interface DiagnosticEvent {
+  time: number;
+  event_properties: {
+    response_error_code: number;
```

Comment:
Should error code and others be enum?

Reply:
Keep status code as number but make messages as constants.